app/Modules/Layouts/Views/layouts/_header.blade.php

```diff
@@ -28,9 +28,9 @@
                         <i class="fa fa-cog"></i>
                     </a>
                     <ul class="dropdown-menu">
-                        <li><a href="{{ route('addons.index') }}">@lang('ip.addons')</a></li>
+                        {{--<li><a href="{{ route('addons.index') }}">@lang('ip.addons')</a></li>--}}

```

Comment:
Why are they commented out?

Reply:
Actually ... the addons.index is working at the moment, so re-committed this file with another commit message. The custom_fields route doesn't work, so i deleted that from the menu: 675265b